<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор приподнятого косинуса</string>
  </property>
  <property name="windowIcon">
   <iconset resource="q.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="optionsGRB">
     <property name="title">
      <string>Параметры фильтра</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="symbolRateLBL">
          <property name="text">
           <string>Символьная скорость (в Гц)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>39</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="symbolRateHSL">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="symbolRateSPB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="rolloffLBL">
          <property name="text">
           <string>Коэффициент сглаживания</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="rolloffHSL">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rolloffDSPB">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="samplingLBL">
          <property name="text">
           <string>Частота дискретизации (в Гц)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>33</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="samplingHSL">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="pageStep">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="samplingSPB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="numOfTapsLBL">
          <property name="text">
           <string>Число коэффициентов</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="numOfTapsHSL">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numOfTapsSPB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="computePBN">
     <property name="text">
      <string>Рассчитать фильтр</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="savePBN">
     <property name="text">
      <string>Сохранить фильтр</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="q.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>symbolRateHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>symbolRateSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>symbolRateSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>symbolRateHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>samplingSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>samplingHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numOfTapsHSL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numOfTapsSPB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numOfTapsSPB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numOfTapsHSL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
